docs/pages/operations-guide.md

```diff
@@ -374,11 +374,9 @@ On the other hand, the secret with path `secret-svc/instance1/Secret_Path2` cann
 
   
 
-### Absolute and Relative File Paths for Secrets
+**Note:** Absolute paths, with leading slash character, are not allowed.   The file paths for secret is relative path to sandbox, the secret file is placed under the sandbox. 
 
```

Comment:
Suggest:

Absolute paths (paths with a leading slash) to secrets are not supported. The secret file is located within the sandbox, therefore the file path for a secret must be relative to the sandbox.
- Supported: `secret-dvc/Secret_Path1`
- Not supported: `/secret-dvc/Secret_Path1`